### PR TITLE
Add basic skeleton loaders to v2 project

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Basic skeleton loaders for:
- Treasury stats
- funding distribution stats

## Screenshots or screen recordings

| loading | done |
| --- | --- |
| Screen Shot 2022-04-01 at 1 44 30 pm | Screen Shot 2022-04-01 at 1 44 42 pm | 

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

**Comment by aeolianeth** [comment]
@johnnyd-eth one for u

File: src/components/v2/V2Project/TreasuryStats.tsx [hunk 1]
```diff
@@ -77,21 +77,22 @@ export default function TreasuryStats() {
...
-          }
-          statValue={
+      <StatLine
+        loading={!Boolean(distributionLimit)}
+        statLabel={<Trans>Distributed</Trans>}
+        statLabelTip={
+          <Trans>
+            The amount that has been distributed from the Juicebox balance in
+            this funding cycle, out of the current funding target. No more than
+            the funding target can be distributed in a single funding cycle—any
```
Comment (johnnyd-eth) [inline src/components/v2/V2Project/TreasuryStats.tsx:90]: 'cycle—any' -> 'cycle — any' ?

**Review by johnnyd-eth — APPROVED** [review]
Gr8 gr8 gr8. Left a single small comment but approving in advance!

File: src/components/v2/V2Project/TreasuryStats.tsx (hunk 1 above)
Comment (aeolianeth) [reply src/components/v2/V2Project/TreasuryStats.tsx:90]: Funnily enough, looks like this is the proper use: https://www.thesaurus.com/e/grammar/em-dash/

I didn't change this though, was already there.

**Pull request merged**